### PR TITLE
chore: add bun to screenpipe

### PR DESCRIPTION
**Pull request by louis030195**

### CODE REVIEW

File: Formula/s/screenpipe.rb [hunk 1]
```diff
@@ -15,6 +15,7 @@ class Screenpipe < Formula
   depends_on "cmake" => :build
   depends_on "pkgconf" => :build
   depends_on "rust" => :build
+  depends_on "bun"
```
Comment (chenrui333) [inline Formula/s/screenpipe.rb:18]: we dont have bun formula

Comment (louis030195) [reply Formula/s/screenpipe.rb:18]: Screenshot 2024-12-09 at 6 40 10 PM

**Review by SMillerDev — CHANGES_REQUESTED** [review]

File: Formula/s/screenpipe.rb [hunk 2]
```diff
@@ -30,13 +32,16 @@ class Screenpipe < Formula
   end
 
   def install
+    system "curl", "-fsSL", "https://bun.sh/install", "|", "bash"
```
Comment (SMillerDev) [inline Formula/s/screenpipe.rb:35]: That's very much disallowed:

https://docs.brew.sh/Acceptable-Formulae#we-dont-like-install-scripts-that-download-unversioned-things

**Review by SMillerDev — CHANGES_REQUESTED** [review]

**Review by SMillerDev — COMMENTED** [review]
If you need bun in your formula it's going to need to be build from source. Please stop trying to evade that requirement.

**Comment by p-linnane** [comment]
No one has been able to get `bun` to build from source to add it to Homebrew: https://github.com/Homebrew/homebrew-core/pulls?q=sort%3Aupdated-desc+is%3Apr+%22bun+new+formula%22

If `screenpipe` now requires it to build then we will most likely need to deprecate `screenpipe` and remove it from Homebrew/core.

**Pull request closed without merging**